sdk/src/main/kotlin/com/processout/sdk/core/logger/SystemLoggerService.kt

```diff
@@ -0,0 +1,29 @@
+package com.processout.sdk.core.logger
+
+import android.util.Log
+
+internal class SystemLoggerService(
+    minimumLevel: LogLevel
+) : BaseLoggerService(minimumLevel) {
+
+    companion object {
+        private const val MAX_LOG_LENGTH = 4000
+    }
+
+    override fun log(event: LogEvent) {
+        val message = "${event.message} ${event.attributes}"
```

Comment:
Would it be possible to rewrite it a bit simpler?
```kotlin
val message = "${event.message} ${event.attributes}"
for (item in message.chunked(MAX_LOG_LENGTH)) {
   // todo: maybe remove trailing newline from item because println will add one on its own
   Log.println(event.level.priority, event.tag, item)
}
```

Reply:
Good point! Updated.

Reply:
No need to trim newline, as Log.println() already handles it and ignores multiple newlines.